config.toml

```diff
@@ -70,7 +81,12 @@ install-fdroid = "Installer Organic Maps depuis F-Droid"
 language = "Français"
 name = "Nom"
 token = "Jeton"
+
+# Language: it (Italian / Italiano)
 [languages.it]
+taxonomies = [
+  {name = "faq", feed = false},
```

Comment:
Thanks!

1. How it will work if every language has a localized name instead of "faq" (e.g. a synonym of "help" for example)?
2. What is generated now with this change in the root FAQ page for translations?

Reply:
> 1. How it will work if every language has a localized name instead of "faq" (e.g. a synonym of "help" for example)?

As far as I understand, this "faq" taxonomy string shall be stated the same here for all languages for the FAQs pages to work. And each language can then state the name of the FAQ taxonomy for every FAQ page.

For example in `content/faq/text-to-speech-android-tts/index.ru.md` file:
```
  taxonomies:
  faq: ["Голосовые подсказки"]
```
I'm no expert in this matter. @strump can most probably enlighten us here...

Reply:
> 2\. What is generated now with this change in the root FAQ page for translations?

There's no change after this. As the matching `content/faq/_index.XX.md` files are still missing, the default English root FAQ page is shown.